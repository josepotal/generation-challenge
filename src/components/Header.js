import React from 'react';

const Header = () => (
  <header className="header">
    <div className="header-title"> Stores Directory </div>
  </header>
);


export default Header;
