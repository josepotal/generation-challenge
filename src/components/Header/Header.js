import React from 'react';
import styles from './Header.module.scss';

const Header = () => (
  <header className={styles.header}>
    <div className={styles.title}> Stores Directory </div>
  </header>
);

export default Header;
