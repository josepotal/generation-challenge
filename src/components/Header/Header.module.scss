@value colors: '../../styles/variables.scss';
@value lightBlue, white from colors;

.header {
  height: 60px;
  background: lightBlue;
  display: flex;
  justify-content: center;
  align-items: center;
}

.title {
  font-size: 1.6em;
  font-weight: bold;
  color: white;
}
