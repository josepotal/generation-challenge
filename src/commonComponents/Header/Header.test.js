import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

test('Header component', () => {
  const component = renderer.create(<Header />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
