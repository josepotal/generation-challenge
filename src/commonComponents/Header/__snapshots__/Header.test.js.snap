// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component 1`] = `
<header>
  <div>
     Stores Directory 
  </div>
</header>
`;
